spec/models/article_spec.rb

```diff
@@ -401,35 +405,40 @@ def build_and_validate_article(*args)
 
   describe "#async_score_calc" do
     context "when published" do
-      before { ActiveJob::Base.queue_adapter = :inline }
```

Comment:
So, `:inline` introduced flakyness in the tests, because once set should be reset to `:test` after each test. It's super easy to forget about resetting, which would trigger a cascade of unpredictable behaviors. There's a better way to tell Rails to execute active jobs inline, which is to use [ActiveJob::TestHelper.perform_enqueued_jobs](https://api.rubyonrails.org/classes/ActiveJob/TestHelper.html#method-i-perform_enqueued_jobs)